include/gridtools/storage/builder.hpp

```diff
@@ -44,7 +44,9 @@ namespace gridtools {
                             return false;
                     return true;
                 };
+#ifdef _OPENMP
```

Comment:
I think that warning against unknown pragmas is stupid.
AFAIK, the standard says that unknown pragmas should be ignored. Why we can not use that guaranty?
Is it possible just to add `-Wno-unknown-pragmas` after `-Wall`?

Reply:
I would propose to be pragmatic:
- It will be in a limited number of places
- Setting `-Wno-unknown-pragmas` would have to be done in each user code, we cannot/should not do it for the user. We will get complaints from users in the future. We even got one from an internal user @mbianco.
- Apparently, the gcc community decided that it's worth warning about that, let's accept the decision.

Reply:
OK, I am convinced. Let us guard all pragmas with the ifdefs